cdap-docs/integrations/source/partners/cloudera/index.rst

```diff
@@ -56,6 +60,12 @@ results.
 
 - |cloudera-ingesting|_ Ingest and explore data using **Impala**
 
+.. |cloudera-navigator| replace:: **Navigator Integration:**
+.. _cloudera-navigator: navigator-integration.html
+
+- |cloudera-navigator|_ Metadata discovery and search using CDAP's **Navigator Integration
```

Comment:
However, "search" and "discovery" are essentially the same thing. Maybe 
``...: Metadata-based discovery of CDAP entities from Cloudera Manager``